dragonchain/transaction_processor/level_5_actions.py

```diff
@@ -107,6 +109,25 @@ def execute() -> None:
     # Only check confirmations if there are any pending
     check_confirmations()
 
+    # go through the failed claims backlog
+    _log.info("Trying to finalize any previously failed claims...")
+    process_claims_backlog()
+
+
+def process_claims_backlog() -> None:
+    claims = redis.hgetall_sync("mq:failed-claims", decode=False)
+    claim_list = []
+    for _, value in claims.items():
+        claim_list.append(value)
+    _log.info(f"Claim backlog count: {len(claim_list)}")
+    for claim_check_id in claim_list:
+        try:
+            matchmaking.resolve_claim_check(claim_check_id)
+        except Exception:
+            _log.exception("Failure to finalize claim in matchmaking.  Skipping the rest of the retry queue.")
+            return  # short-circuit and exit early, matchmaking still unreachable
+        redis.hdel_sync("mq:failed-claims", claim_check_id)  # claim successfully finalized, remove from backlog
```

Comment:
hdel? that's definitely not right for a redis list.

Reply:
Should be using set.  Fixed.